docs/content/preview/yugabyte-voyager/migrate/migrate-steps.md

```diff
@@ -22,7 +22,7 @@ The following page describes the steps to perform and verify a successful offlin
 
 | Phase | Step | Description |
 | :---- | :--- | :---------- |
-| PREPARE | [Install voyager](../../install-yb-voyager/#install-yb-voyager) | yb-voyager supports RHEL, CentOS, Ubuntu, and macOS, as well as airgapped and Docker-based installations. |
+| PREPARE | [Install voyager](../../install-yb-voyager/#install-yb-voyager) | Voyager supports RHEL, CentOS, Ubuntu, and macOS. It can also be installed as airgapped and on Docker. |
```

Comment:
I think we should include the steps "Create an export directory" and "Set up a configuration file" in the PREPARE section of the table. These two steps are missing.

Reply:
I will let @ShivanshGahlot, take a call here!